hail/python/hail/methods/qc.py

```diff
@@ -491,13 +504,128 @@ def n_discordant(counter):
     return glob, per_sample.cols(), per_variant.rows()
 
 
+def _service_vep(ht, config, block_size, csq, tolerate_parse_error):
+    token = secret_alnum_string(16)
+
+    backend = hl.context.current_backend()
+    assert isinstance(backend, hl.backend.service_backend.ServiceBackend)
```

Comment:
Take the backend as an argument with the right type. Then in the caller do this:
```
backend = hl.current_backend()
if isinstance(backend, hl.backend.ServiceBackend):
    _service_vep(..., backend)
else:
    ...
```

This way we do one is instance check. If we try to pass `hl.current_backend` then mypy will complain because the types don't match.